### PR TITLE
Doc 348: grid docs bug bash byoc

### PR DESCRIPTION
**Pull request by dmitsf**
# What does this PR do?

Docs fixes for BYOC.

### CODE REVIEW

File: docs/features/datastores/overview.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Demystifying Datastores
+# Demystifying Datastorses
```
Comment (alexandercort) [inline docs/features/datastores/overview.md]: @oojo12 spelling error

**Comment by oojo12** [comment]
Errors from local build @dmitsf 

```
ERROR in ./docs/platform/2_Custom Cloud Credentials/3_adding-custom-cloud-credentials.md

Module build failed (from ./node_modules/@docusaurus/mdx-loader/lib/index.js):
Error: Image static/static/images/platform/IAM.png used in docs/platform/2_Custom Cloud Credentials/3_adding-custom-cloud-credentials.md not found.
    at findImage (C:\Users\Olufemi Ojo\Documents\grid-docs\node_modules\@docusaurus\mdx-loader\lib\remark\transformImage\index.js:46:11)
    at async processImageNode (C:\Users\Olufemi Ojo\Documents\grid-docs\node_modules\@docusaurus\mdx-loader\lib\remark\transformImage\index.js:68:27)
    at async Promise.all (index 0)
    at async transformer (C:\Users\Olufemi Ojo\Documents\grid-docs\node_modules\@docusaurus\mdx-loader\lib\remark\transformImage\index.js:86:9)


ERROR in ./docs/platform/2_Custom Cloud Credentials/4_customer-managed-byoc.md

Module build failed (from ./node_modules/@docusaurus/mdx-loader/lib/index.js):
Error: Image static/static/images/platform/IAM.png used in docs/platform/2_Custom Cloud Credentials/4_customer-managed-byoc.md not found.
    at findImage (C:\Users\Olufemi Ojo\Documents\grid-docs\node_modules\@docusaurus\mdx-loader\lib\remark\transformImage\index.js:46:11)
    at async processImageNode (C:\Users\Olufemi Ojo\Documents\grid-docs\node_modules\@docusaurus\mdx-loader\lib\remark\transformImage\index.js:68:27)
    at async Promise.all (index 0)
    at async transformer (C:\Users\Olufemi Ojo\Documents\grid-docs\node_modules\@docusaurus\mdx-loader\lib\remark\transformImage\index.js:86:9)
```

**Comment by oojo12** [comment]
@dmitsf have you confirmed all the BYOC steps are correct and no content is missing? I think there was also a conversation between you me and Robert about missing dependencies. Can you add them here?

**Comment by oojo12** [comment]
After I can approve. The prereq link should be https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html. Didn't commit myself since I didn't want to cause some merge conflicts for you incase there was more stuff to be changed.

**Comment by dmitsf** [comment]
> After I can approve. The AWS CLI prereq link should be https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html. Didn't commit myself since I didn't want to cause some merge conflicts for you incase there was more stuff to be changed.

Thanks, Femi! Added. About aws-iam-authenticator, we don't need it for aws-cli >= 2

**Review by oojo12 — APPROVED** [review]
Thanks for the hard work and fixes here @dmitsf

**Review by essiequoi — APPROVED** [review]

**Review by essiequoi — APPROVED** [review]

**Pull request merged**